app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(
+            self,
+            name: str,
+            age: int
+    ) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"])
+                   for person in people]
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife\
+                = Person.people[person["wife"]]
+        elif person.get("husband"):
+            Person.people[person["name"]].husband\
+                = Person.people[person["husband"]]
+    return person_list
```

Comment:
Try to avoid line breaks where possible to improve the code style. You could allocate a separate variable at the beginning of the cycle to get the required `Person` class instance in advance, e.g.:
```suggestion
    for person in people:
        person_instance = Person.people[person["name"]]
        
        if person.get("wife"):
            person_instance.wife = Person.people[person["wife"]]
        elif person.get("husband"):
            person_instance.husband = Person.people[person["husband"]]
    
    return person_list
```